DMFT/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.data.mongodb.host=localhost
```

Comment:
Is this file configuration by convention?  I don't see how it's referenced in the other code.

Reply:
This is the environment variables file in Spring Boot (.env equivalent).